sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCoalesceBatches.scala

```diff
@@ -608,13 +608,15 @@ case class GpuCoalesceBatches(child: SparkPlan, goal: CoalesceGoal)
     // cache in local vars to avoid serializing the plan
     val outputSchema = schema
     val decompressMemoryTarget = maxDecompressBatchMemory
-    val cannotSpill = child.schema.fields.exists { f =>
-      f.dataType match {
-        case MapType(_, _, _) | ArrayType(_, _) | StructType(_) => true
-        case _ => false
-      }
-    }
-
+    // disabling spillable batches due to
+    // https://github.com/rapidsai/cudf/issues/7514
+    val cannotSpill = true
```

Comment:
we can also consider leaving the code in but adding a new conf like "disable.spillable.batches" . We would flip the default to false once the bug is fixed.

Reply:
IMO configs are overkill for this.  Bugfix is already posted in cudf-0.19, will definitely be fixed in 0.5.